generators/server/templates/src/main/docker/cassandra/scripts/_autoMigrate.sh

```diff
@@ -10,12 +10,16 @@ function log {
     echo "[$(date)]: $*"
 }
 
+function logDebug {
+    ((DEBUG_LOG)) && echo "[DEBUG][$(date)]: $*"
+}
+
 function waitForClusterConnection() {
     retryCount=0
     maxRetry=20
     cqlsh -e "Describe KEYSPACES;" $CASSANDRA_CONTACT_POINT &>/dev/null
     while [ $? -ne 0 ] && [ "$retryCount" -ne "$maxRetry" ]; do
-        log 'cassandra not reachable yet. sleep and retry. retryCount =' $retryCount
+        logDebug 'cassandra not reachable yet. sleep and retry. retryCount =' $retryCount
```

Comment:
on local machine, if user doesn't have cqlsh, the script won't work and there is no message to warn user
=> anyway, it's not so important, it's supposed to be used with docker, right ? :D